doc/redirects.json

```diff
@@ -2718,29 +2718,20 @@
     "index.html#ssec-language-go"
   ],
   "buildgomodule-parameters": [
-    "index.html#buildgomodule-parameters"
+    "index.html#buildgomodule-parameters",
+    "index.html#ssec-go-common-attributes"
   ],
   "ex-buildGoModule": [
     "index.html#ex-buildGoModule"
   ],
-  "buildGoModule-vendorHash": [
```

Comment:
Shouldnt this redirect to the new urls?

Reply:
Yeah, that's hard to see because the identifier moved to a different line. This should be solved when the helper command is made.